core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -146,6 +146,16 @@ oppia.controller('ExplorationFeedback', [
     });
   };
 
+  $scope.isSendDisabled = function(updatedStatus) {
```

Comment:
Prefer: isSendButtonDisabled

Also, I think for parity we should pass along $parent.$parent.newMessageText as well, just like we do in addMessage. It seems odd to pass one of these and not the other.